src/backend/utils/fmgr/dfmgr.c

```diff
@@ -297,6 +298,13 @@ internal_load_library(const char *libname)
 		file_tail = file_scanner;
 	}
 
+	if (file_exists("data_encryption.key") && file_read_buffer_modify_hook == NULL)
+	{
+		ereport(ERROR,
+				(errmsg("this cluster has encryption enabled, but necessary extensions does not loaded"),
+				 errhint("add gp_data_encryption to shared_preload_libraries, in file postgresql.conf")));
```

Comment:
`errhint("add ‘gp_data_encryption’ to ‘shared_preload_libraries’, in file ‘postgresql.conf’")));`

Add quote `''` or `""` to the terms.